docs/UserGuide.md

```diff
@@ -283,210 +341,357 @@ Format: `edit INDEX [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​`
+### View help: `help` <a name="help"></a>
 
-### Viewing help: `help`
 Shows a message explaining how to access the User Guide or obtain in-app guidance for selected commands.
 
-Format: `help`
-
+<div markdown="block" class="alert alert-primary">
+**:mag_right: Format:**
+`help`
 ![result for 'help'](images/HelpBox.png)
+</div>
 
 <br>
 
-### Viewing in-app guidance: `help COMMAND`
+### View in-app guidance: `help COMMAND` <a name="help_command"></a>
 
 Displays concise instructions for users to learn the given `COMMAND`'s function.
 
-Format: `help COMMAND`
+<div markdown="block" class="alert alert-primary">
+**:mag_right: Format:**
+`help COMMAND`
 * Acceptable keywords to use in `COMMAND`: `add` `delete` `edit` `find` `remark` `share` `picture` `pin` `unpin`
+</div>
 
-Examples:
-* `help add` displays a concise guide on how to use the `add` command.
-* `help edit` displays a concise guide on how to use the `edit` command.
-
-  ![result for 'help'](images/HelpAdd.png)
-
-<br>
+<div markdown="block" class="alert alert-success">
+**:heavy_check_mark: Example:** `help add` 
 
-### Remove all data: `clear`
-Removes all contacts in the app.
+Displays a concise guide on how to use the `add` command.
+![result for 'help'](images/HelpAdd.png)
+</div>
 
-Format: `clear`
+<div markdown="block" class="alert alert-success">
+**:heavy_check_mark: Example:** `help edit` 
 
-<div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-Please be mindful that entering this command will wipe out all your contacts stored in SociaLite!
+Displays a concise guide on how to use the `edit` command.
 </div>
 
 <br>
 
-### Exiting the program : `exit`
+### Exit the program : `exit` <a name="exit"></a>
 
 Exits the program.
 
-Format: `exit`
+<div markdown="block" class="alert alert-primary">
+**:mag_right: Format:**
+`exit`
+</div>
 
 <br>
 
-### Saving the data
-
-SociaLite data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+--------------------------------------------------------------------------------------------------------------------
 
-<br>
+## FAQ <a name="faq"></a>
 
-### Editing the data file
+**Q**: How is my data saved?<br>
+**A**: SociaLite data are saved in the hard disk automatically after any command that alters the data. There is no need to save manually.
 
-SociaLite data are saved as a JSON file `[JAR file location]/data/SociaLite.json`. Advanced users are welcome to update data directly by editing that data file.
+**Q**: How do I transfer my data to another Computer?<br>
+**A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous SociaLite home folder.
 
+**Q**: Can I edit my data?<br>
+**A**: SociaLite data are saved as a JSON file `[JAR file location]/data/SociaLite.json`. Advanced users are welcome to update data directly by editing that data file.
```

Comment:
Extremely small nit here, but I think perhaps can consider using `<JAR file location>` instead! Since the square bracket is similar to the usage in standard notations, the users might get confused and thought that this is an optional field.

Reply:
Change reflected in line 669